channel/fd_channel.cpp

```diff
@@ -117,6 +155,9 @@ void FdChannel::SetupForkServer(char *const pargv[]) {
         dup2(null_fd, STDOUT_FILENO);
         dup2(null_fd, STDERR_FILENO);
 
+        // Disable AFL's forkserver
+        setenv("__AFL_DEFER_FORKSRV", "1", 1);
+
```

Comment:
FYI, __AFL_DEFER_FORKSRV is not for disabling fork servers. It's jut "defer" the start of fork servers.

Reply:
Thanks for supplement. I removed this code.